nautobot/core/management/commands/generate_test_data.py

```diff
@@ -278,7 +278,7 @@ def _create_batch(some_factory, count, description="", **kwargs):
         # _create_batch(ClusterFactory, 10)
         # _create_batch(VirtualMachineFactory, 10)
         # We need to remove them from there and enable them here instead, but that will require many test updates.
-        _create_batch(StaticGroupFactory, 20)
+        _create_batch(DynamicGroupFactory, 20, description="and StaticGroupAssociations")
```

Comment:
Is this only creating dynamic groups with type static? Should the description reflect that?

Reply:
The factory currently makes TYPE_STATIC and TYPE_DYNAMIC_FILTER groups but not yet TYPE_DYNAMIC_SET.